lineman/app/components/notification_volume/notification_volume_dropdown.haml

```diff
@@ -1,4 +1,4 @@
-.notification-volume{dropdown: ''}
+.notification-volume{ng-show: 'canChangeVolume()', dropdown: ''}
```

Comment:
(You likely know this already but..)

ng-if doesn't render the inner block into the DOM at all until it is true, which means any directives, watchers, and nested scopes aren't part of the page. However it takes more time to render and digest all that stuff every time the value changes, so it's better to avoid ng-if if the value is likely to change often.
ng-show just slaps a 'display: none' style on the element when it's false, but still renders and watches for changes to all of the sub elements. It's super low-overhead to switch on and off, though.